more transitions

natec0923 opened the pull request:
added drive state machine transitions (balance, follow tag), cleaned up some imports (to be more organized), added the score state where we use dpad to shift to different nodes (not perfect, just baseline)

natec0923 committed:
more transitions

ChristianPawlowicz commented on src/main/java/frc/robot/subsystems/drive/DrivetrainStateMachine.java:
```diff
@@ -10,14 +10,24 @@
-        Supplier<Boolean> checkAButton = () -> {
+
+
+       /*  Supplier<Boolean> checkAButton = () -> {
```
Remove Code that we do not use anymore, do not just comment it out. We have githug for looking at file history. No need to keep a comment record. It just makes the code less clean.

ChristianPawlowicz commented:
> added drive state machine transitions (balance, follow tag), cleaned up some imports (to be more organized), added the score state where we use dpad to shift to different nodes (not perfect, just baseline) 

Good description!

natec0923 committed:
updated transitions

natec0923 commented:
Updated Transitions: Removed old transition requirements from scoring positions to open claw - changed to another button (b/c in the old code we would just be pushing the buttons again to open the claw). Mapped all the buttons for manipulator and driver. Added labels to the idle state transitions b/c it gave me a headache.

katiewang22 committed:
Merge branch 'main' into MoreTransitionsNate

katiewang22 committed:
Fix merge conflicts
